csm/conf/setup.py

```diff
@@ -530,18 +532,22 @@ def _get_data_nw_info(minion_id):
         return data_nw
 
     @staticmethod
-    def _set_db_host_addr(backend, addr):
+    def _set_db_host_addr_list(backend, addr_list):
         """
-        Sets database backend host address in CSM config.
+        Sets database backend host addresses in CSM config.
 
         :param backend: Databased backend. Supports Elasticsearch('es'), Consul ('consul').
-        :param addr: Host address.
+        :param addr_list: Host addresses list.
         """
+
         if backend not in ('es', 'consul'):
-            raise CsmSetupError(f'Invalid database backend "{addr}"')
-        key = f'databases.{backend}_db.config.host'
+            raise CsmSetupError(f'Invalid database backend "{addr_list}"')
```

Comment:
Oops, I found an error in the original implementation :) That should be `backend` instead of `addr_list`.

Can you fix it, please?

Reply:
Done